style/article-2022.css

```diff
@@ -895,7 +895,25 @@ small {
 	margin: 0;
 	}
 
+/* **************character markup******* */
+    .codepoint bdi {
+        line-height: 1em;
+        font-size: 1.25em;
+        padding-inline: 0.25em;
+    }
+	
+    .codepoint img {
+        height: 2em;
+        margin-inline: 0.25em;
+        vertical-align: bottom;
+    }
 
+    .uname {
```

Comment:
If we add `.uname` here, we should probably remove the `.uname` below: https://github.com/w3c/i18n-drafts/blob/d66cd317fd88a414a579c5b6fe3c8ef2046ec2d3/style/article-2022.css#L993-L996

Reply:
Done.